bff/pkg/server/promql_codegen_test_cases.json

```diff
@@ -132,8 +115,8 @@
       "limit": "",
       "offset": ""
     },
-    "expected": "sort_desc(avg_over_time(count by(datname, usename) (label_replace(label_replace(cc_pg_stat_activity{datname=~\"db1\"}, \"datname\", \"/*pg internal*/\", \"datname\", \"\"), \"usename\", \"/*pg internal*/\", \"usename\", \"\"))[3600s:]))",
-    "has_error": false
+    "expected": "sort_desc(avg_over_time(count by(datname, usename) (label_replace(label_replace(cc_pg_stat_activity{datname=~\"db1\"}, \"datname\", \"/*pg internal*/\", \"datname\", \"\"), \"usename\", \"/*pg internal*/\", \"usename\", \"\"))[-9223372036s:]))",
```

Comment:
"-9223372036s" doesn't seem right. It should be `3600s`. This shouldn't have an error. If end is not specified, `now` should be used by default.